components/web/aget/patches/01_Makefile.Solaris.patch

```diff
@@ -0,0 +1,29 @@
+--- aget-0.4.1/Makefile.Solaris		2016-12-21 22:44:23.411350275 -0200
++++ aget-0.4.1-oi/Makefile.Solaris	2016-12-21 22:49:04.283929368 -0200
+@@ -2,19 +2,21 @@
+ # http://www.enderunix.org/aget/
+ 
+ OBJS = main.o Aget.o Misc.o Head.o Signal.o Download.o Resume.o
+-CFLAGS  = -g -W -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
+-LDFLAGS = -lpthread -lsocket -lnsl -lresolv
+-CC = gcc
++CFLAGS  += -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64
++LDFLAGS += -lpthread -lsocket -lnsl -lresolv
+ STRIP=/usr/ccs/bin/strip
+ 
+ all: $(OBJS)
+-	$(CC) -o aget $(OBJS) $(LDFLAGS)
++	$(CC) -m$(BITS) -o aget $(OBJS) $(LDFLAGS)
+ 
+ strip: $(all)
+ 	$(STRIP) aget
+ 	
+ install:
+-	cp -f aget /usr/local/bin/aget
```

Comment:
Actually the build is running for 32 and 64bit. That's right. I have nothing to correct.
Buid output:

make[1]: Entering directory '/home/fronald/Projects/oi-userland/components/web/aget/build/i86'
/usr/gcc/4.9/bin/gcc -m32 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o main.o main.c
/usr/gcc/4.9/bin/gcc -m32 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Aget.o Aget.c
/usr/gcc/4.9/bin/gcc -m32 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Misc.o Misc.c
/usr/gcc/4.9/bin/gcc -m32 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Head.o Head.c
/usr/gcc/4.9/bin/gcc -m32 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Signal.o Signal.c
/usr/gcc/4.9/bin/gcc -m32 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Download.o Download.c
/usr/gcc/4.9/bin/gcc -m32 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Resume.o Resume.c
/usr/gcc/4.9/bin/gcc -m32 -o aget main.o Aget.o Misc.o Head.o Signal.o Download.o Resume.o -lpthread -lsocket -lnsl -lresolv
make[1]: Leaving directory '/home/fronald/Projects/oi-userland/components/web/aget/build/i86'
/usr/bin/touch /home/fronald/Projects/oi-userland/components/web/aget/build/i86/.built
/bin/rm -f -r /home/fronald/Projects/oi-userland/components/web/aget/build/amd64 ; /bin/mkdir -p /home/fronald/Projects/oi-userland/components/web/aget/build/amd64
/home/fronald/Projects/oi-userland/tools/cloney /home/fronald/Projects/oi-userland/components/web/aget/aget-0.4.1 /home/fronald/Projects/oi-userland/components/web/aget/build/amd64
symlink cloning /home/fronald/Projects/oi-userland/components/web/aget/aget-0.4.1 to /home/fronald/Projects/oi-userland/components/web/aget/build/amd64
(cd /home/fronald/Projects/oi-userland/components/web/aget/build/amd64; /usr/bin/env LD_OPTIONS="-M /usr/lib/ld/map.noexstk -M /usr/lib/ld/map.noexdata -M /usr/lib/ld/map.pagealign -Bdirect -z ignore" LD_EXEC_OPTIONS="-z aslr=disable"  BITS=64 CC=/usr/gcc/4.9/bin/gcc CFLAGS="-m64 -O3 " /usr/gnu/bin/make   )
make[1]: Entering directory '/home/fronald/Projects/oi-userland/components/web/aget/build/amd64'
/usr/gcc/4.9/bin/gcc -m64 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o main.o main.c
/usr/gcc/4.9/bin/gcc -m64 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Aget.o Aget.c
/usr/gcc/4.9/bin/gcc -m64 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Misc.o Misc.c
/usr/gcc/4.9/bin/gcc -m64 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Head.o Head.c
/usr/gcc/4.9/bin/gcc -m64 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Signal.o Signal.c
/usr/gcc/4.9/bin/gcc -m64 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Download.o Download.c
/usr/gcc/4.9/bin/gcc -m64 -O3  -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c -o Resume.o Resume.c
/usr/gcc/4.9/bin/gcc -m64 -o aget main.o Aget.o Misc.o Head.o Signal.o Download.o Resume.o -lpthread -lsocket -lnsl -lresolv
make[1]: Leaving directory '/home/fronald/Projects/oi-userland/components/web/aget/build/amd64'
/usr/bin/touch /home/fronald/Projects/oi-userland/components/web/aget/build/amd64/.built
```